zerver/views/realm_linkifiers.py

```diff
@@ -43,3 +43,26 @@ def delete_linkifier(
     except RealmFilter.DoesNotExist:
         return json_error(_("Filter not found"))
     return json_success()
+
+
+@require_realm_admin
+@has_request_variables
+def update_linkifier(
+    request: HttpRequest,
+    user_profile: UserProfile,
+    filter_id: int,
+    pattern: str = REQ(),
+    url_format_string: str = REQ(),
+) -> HttpResponse:
+    try:
+        do_update_linkifier(
+            realm=user_profile.realm,
+            id=filter_id,
+            pattern=pattern,
+            url_format_string=url_format_string,
+        )
+        return json_success()
+    except RealmFilter.DoesNotExist:
+        return json_error(_("Filter not found"))
```

Comment:
This should be `Linkifier not found` 😅

Reply:
Oh, I see you've fixed this in a later commit in this PR. But why introduce it in the first place?

Reply:
Yeah, my point was, you can change just the `delete_linkifier` code in a commit at the beginning of this PR, and then just write this new code correctly.